.github/workflows/build.yaml

```
@@ -54,6 +54,13 @@ jobs:
       - name: Build PHP
         run: nix-build -A outputs.checks.x86_64-linux.${{ steps.params.outputs.attr }}-php
 
+      - name: List extensions
+        run: |
+          NIXPKGS_ALLOW_BROKEN=1 \
+          NIXPKGS_ALLOW_INSECURE=1 \
+          NIXPKGS_ALLOW_UNFREE=1 \
+          nix eval --json --impure --expr '(import ./.).packages.x86_64-linux.${{ steps.params.outputs.attr }}.extensions'
```

With the following, you do not need to care about what extension status:

```suggestion
          nix eval --json --impure --expr 'builtins.attrNames (import ./.).packages.x86_64-linux.${{ steps.params.outputs.attr }}.extensions'
```